### PR TITLE
Mooks! (Inquisitor's Thugs)

### PR DESCRIPTION
**Pull request by Plutolover99**
## About The Pull Request
Have you ever thought that the Inquisitor was... Kind of weak? He's a man who's out of place, without most friends due to the fact that the Church is an unreliable ally against the heretical threats. This seeks to change that.

It adds a role called the Mook! Mooks are lowlife thugs that are apart of the inquisition to help the Inquisitor. They aren't specialized for combat, moreso shaking down civilians, harrassing people, and sneaking around in places they shouldn't. They serve as the muscle for the Inquisitor when he decides to knock on your door, or he sneaks into places he shouldn't to plant evidence for a conviction.

I'd like to offer Zakanater a MASSIVE thank you for his assistance, he is responsible for the testing and a large part of the mapping that made this role possible.

## Why It's Good For The Game
A good, combat-adjacent role that has good utility and a neat way to RP. They're loyal to the Inquisitor, so he can actually be more investigative without worrying of getting his shit pushed in the first mistake that happens.

This is also technically a Church buff as well, adding some muscle to the Church's arsenal should the Keep ever decide that it wants to go on another massacre, these 3 should at least make it a little harder.

### CODE REVIEW

**Comment by MefAnneFeminene** [comment]
I want this test-merged. Please, this is a good idea. You guys will love it.

**Comment by SixplyDev** [comment]
Seems like a good role, my only concern is that they may be more loyal to the Church than the Inquisitor. I suppose that's a player issue though, three thugs to help the Inquisitor is an excellent idea. I do think that perhaps giving the inquisitor maybe some light armor (an arming shirt or something similar) may help during lowpop though. I love this PR and I love this idea, it should hopefully give the Inquisitor more leverage when dealing with an antagonistic Church or uncooperative folks.

File: code/modules/jobs/job_types/roguetown/church/confessor.dm [hunk 1]
```diff
@@ -1,59 +1,68 @@
...
+	spawn_positions = 3
 
-	allowed_races = RACES_TOLERATED_UP
-	allowed_sexes = list(MALE)
-	tutorial = "Confessors are shady agents of the church hired to spy on the populace and keep them moral. As the most fanatical members of the clergy, their main concern is assisting the local Puritan with their work in extracting confessions of sin as well as hunting night beasts and cultists that hide in plain sight."
+	allowed_races = RACES_VERY_SHUNNED_UP
+	allowed_sexes = list(MALE, FEMALE)
+	allowed_patrons = list(
+		/datum/patron/old_god,
+		ALL_DIVINE_PATRONS,
```
Comment (Cyprex) [inline code/modules/jobs/job_types/roguetown/church/confessor.dm:13]: So everything except psydon? Or everything except inhumen?

**Comment by Cyprex** [comment]
So, these get bad gear, mediocre stats, and good skills(Very similiar to what a rogue gets. We'll see how this goes, I personally don't like the inquisitor by design but I'm testmerging this

**Comment by Cyprex** [comment]
Testmerged

**Comment by HissyNays** [comment]
> They aren't specialized for combat


Okay but they get one of the rarest weapon skills in game, expert in daggers. That alone is a reason to pick rogue adventurer. Simply put, this is too good for a role thats supposed mediocre in combat at max.
You'll need to nerf the daggers skill to 3 to keep it in line with other roles.

**Comment by HissyNays** [comment]
> this is literally just a thief, why does the inquisitor get three thieves, wtf lmfao



> this is literally just a thief, why does the inquisitor get three thieves, wtf lmfao



> this is literally just a thief, why does the inquisitor get three thieves, wtf lmfao

Thief doesn't have so much fighting potential ontop of many abilities like this role.
The lack of armor is very easily solvable and that extra 1 in intellect will allow you to get cloth or leather armor very fast if you know what you're doing. If you see a mook with a shield, basicly run, that expert in dagger will allow them to stab your eyes out very quickly and shield covers for lack dodge expert that thief usually had.
I wish watchmen were as close to any good as this

**Comment by Cyprex** [comment]
Untestmerged until skills are adjusted due to the above reasons- their skills are too strong.

**Comment by radarseas** [comment]
A better idea perhaps, a bit of a bandit lite without the jack of all trades and more skills towards incapitation weaponry like crudgeons or whatever, keep em with the master sneaking and make em dumber and slower so they're not exactly much smarter than thugs and rely on social deception, numbers and catching people off-guard.

_Maybe give em decieving meekness too that currently maniac has, so they appear weak and catch you off-guard._ Give these guys a proper reasoning to be feared but also hidden amonst the town's own and give them the ability to fake a job title perhaps too, so they can pretend to be a harmless cheesemaker or a woodcutter to interrogate you without the pressure of knowing its someone loyal to the inquisition.

The idea of a deceptive ambush role that pretends to be your friend or an average townie to get information and then gangs on you with their other pals when you go somewhere quiet where nobody will believe you or hear you scream is a _lot more thematic_ than zeolot that stabs you and steals your money.

**Comment by MefAnneFeminene** [comment]
Thank you all for the feedback. I'll be adjusting the stats and skills accordingly.

> A better idea perhaps, a bit of a bandit lite without the jack of all trades and more skills towards incapitation weaponry like crudgeons or whatever, keep em with the master sneaking and make em dumber and slower so they're not exactly much smarter than thugs and rely on social deception, numbers and catching people off-guard.
> 
> _Maybe give em decieving meekness too that currently maniac has, so they appear weak and catch you off-guard._ Give these guys a proper reasoning to be feared but also hidden amonst the town's own and give them the ability to fake a job title perhaps too, so they can pretend to be a harmless cheesemaker or a woodcutter to interrogate you without the pressure of knowing its someone loyal to the inquisition only to backstab you later.
> 
> The idea of a deceptive ambush role that pretends to be your friend or an average townie to get information and then gangs on you with their other pals when you go somewhere quiet where nobody will believe you or hear you scream is a _lot more thematic_ than zeolot that stabs you and steals your money.

I really like this idea, and I'll be adjusting their stats to align with this. I'll also give them the Maniac trait you mentioned because that sounds perfect for the role.

From what people have mentioned, I'll be lowering their pickpocketing and dagger stats, but keep their sneaking and raise their cudgel aligned skills accordingly. I'll also give them a chain to apprehend people with.

**Comment by MefAnneFeminene** [comment]
A lot of changes from previous Mooks. Please take a look and give your opinions, and TM this again when you get the chance. @Cyprex @radarseas @Dedwolf @HissyNays

File: code/modules/jobs/job_types/roguetown/church/confessor.dm [hunk 2]
```diff
@@ -1,59 +1,72 @@
...
 	faction = "Station"
-	total_positions = 0
-	spawn_positions = 0
+	total_positions = 3
+	spawn_positions = 3
 
-	allowed_races = RACES_TOLERATED_UP
-	allowed_sexes = list(MALE)
-	tutorial = "Confessors are shady agents of the church hired to spy on the populace and keep them moral. As the most fanatical members of the clergy, their main concern is assisting the local Puritan with their work in extracting confessions of sin as well as hunting night beasts and cultists that hide in plain sight."
+	allowed_races = RACES_VERY_SHUNNED_UP
```
Comment (Jackofashepard) [inline code/modules/jobs/job_types/roguetown/church/confessor.dm:9]: I'm not sure if this was an oversight but why can the mooks be literally everything. This is the psydonite church we are talking about. Shouldnt it be RACES_RESPECTED or RACES_TOLERATED_UP at the very least?

Comment (MefAnneFeminene) [reply code/modules/jobs/job_types/roguetown/church/confessor.dm:9]: Mooks are picked from the lowlifes of society. They aren't respected people. Not only to shunned races get a limited amount of roles, but it'd make sense that the lowlife and expendable agents of Psydon be made up of Shunned Races. These men **serve** the Inquisitor, and the Inquisitor should have little respect for them.

Comment (MefAnneFeminene) [reply code/modules/jobs/job_types/roguetown/church/confessor.dm:9]: But if your dumbfuck PR gets merged I'll change it to be more in line with your flaccid lore.

Comment (radarseas) [reply code/modules/jobs/job_types/roguetown/church/confessor.dm:9]: I think its best they remain at least, tolerated. Especally with the religious paranoia and fervor inquisitors are apparently known for in lore, they wouldn't want species _spawned from or that probably serve the inhuman from their reputation_, disposable sure but would you trust their _loyalty enough to handle your dirty work under psydon's guidance?_

**Comment by MefAnneFeminene** [comment]
@Jackofashepard @radarseas Changed the allowed races to Tolerated and up.

**Pull request closed without merging**